xcore/interface/stitcher.h

```diff
@@ -360,6 +360,21 @@ class BowlModel {
     float             _max_topview_length_mm;
 };
 
+class CubeMapModel {
+public:
+    typedef std::vector<PointFloat2> PointMap;
+
+public:
+    CubeMapModel (const uint32_t image_width, const uint32_t image_height);
```

Comment:
do we need to declare a **const** type here?

Reply:
Removed `const` from this declaration